app/src/main/java/com/example/androidstudies/presenter/ListPresenter.java

```diff
@@ -0,0 +1,67 @@
+package com.example.androidstudies.presenter;
+
+import com.example.androidstudies.view.IListRepository;
+import com.example.androidstudies.view.ListAdapter;
+import com.example.androidstudies.model.Repository;
+
+import java.util.List;
+
+public class ListPresenter implements IListRepository.Presenter {
+    private List<Repository> mListRepositories;
+    private IListRepository.View mView;
+
+    public ListPresenter(){
+    }
+
+    public void onBindViewHolderListPresenter(ListAdapter.MyViewHolder holder, int position){
+        holder.rir.title.setText(mListRepositories.get(position).getRepoName());
+        holder.rir.description.setText(mListRepositories.get(position).getDescription());
+        holder.rir.username.setText(mListRepositories.get(position).getUserName());
+        holder.rir.fullname.setText(mListRepositories.get(position).getUserFullName());
+
+    }
+
+    public int getListRepositoriesSize(){
+        return mListRepositories.size();
+    }
+
+    @Override
+    public void onLoad(List<Repository> listRepositories) {
```

Comment:
onLoad signature shouldn't receive the data, since the view couldn't have the data yet. On here you should call the createRepository method